recipes/libsecret/all/conanfile.py

```diff
@@ -24,24 +25,27 @@ class LibsecretConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "crypto": [False, "libgcrypt", "gnutls"],
         "with_libgcrypt": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
+        "crypto": "libgcrypt",
         "with_libgcrypt": True,
```

Comment:
Please, deprecate `with_libgcrypt` properly. It's duplicated with `crypto`. For older versions, you can raise an Invalid configuration for crypto=gnutls

Reply:
Done.